cms/apps/media/widgets.py

```diff
@@ -0,0 +1,24 @@
+from django import forms
+from django.utils.html import mark_safe
+from django.contrib.admin.widgets import ForeignKeyRawIdWidget
+from django.utils.html import format_html
+
+class ImageThumbnailWidget(ForeignKeyRawIdWidget):
+    '''
+    A widget used to display a thumbnail image preview
+    '''
+    def render(self, name, value, attrs=None):
+        # Import here to avoid circular dependencies
+        from .models import File
+        output = []
+
+        if value:
+            file_obj = File.objects.filter(pk=value).first()
```

Comment:
`File.objects.get(pk=value)` makes more sense - by definition, a PK can never be duplicated and database constraints ensure that `.get` cannot throw an exception.

Reply:
I put this as I wasn't sure if `value` could be unset, and thus we'd need to catch the case for empty fields (hence the if after this as well) but now I look at it it's guarded

Reply:
This has now been replaced with the simplified method above.